src/SprykerEco/Zed/Payone/Business/Payment/PayoneLogsReceiver.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment;
+
+use Generated\Shared\Transfer\PayonePaymentLogCollectionTransfer;
+use Generated\Shared\Transfer\PayonePaymentLogTransfer;
+use SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface;
+
+class PayoneLogsReceiver implements PayoneLogsReceiverInterface
+{
+    public const LOG_TYPE_API_LOG = 'SpyPaymentPayoneApiLog';
+    public const LOG_TYPE_TRANSACTION_STATUS_LOG = 'SpyPaymentPayoneTransactionStatusLog';
+
+    /**
+     * @var \SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface
+     */
+    protected $queryContainer;
+
+    /**
+     * @param \SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface $queryContainer
+     */
+    public function __construct(PayoneQueryContainerInterface $queryContainer)
+    {
+        $this->queryContainer = $queryContainer;
+    }
+
+    /**
+     * Gets payment logs (both api and transaction status) for specific orders in chronological order.
+     *
+     * @param \Propel\Runtime\Collection\ObjectCollection|\ArrayObject $orders
+     *
+     * @return \Generated\Shared\Transfer\PayonePaymentLogCollectionTransfer
+     */
+    public function getPaymentLogs($orders): PayonePaymentLogCollectionTransfer
```

Comment:
* Move this method to a `\SprykerEco\Zed\Payone\Persistence\PayoneRepository`.

Reply:
++